docs/reference/cluster/cluster-info.asciidoc

```diff
@@ -207,14 +207,14 @@ pipeline.
 (integer)
 Total number of failed operations for the ingest pipeline.
 
-`ingested_in_bytes`::
+`ingested_as_first_pipeline_in_bytes`::
 (Optional, integer)
 Total number of bytes of all documents ingested by the pipeline.
 This field is only present on pipelines which are the first to process a document.
 Thus, it is not present on pipelines which only serve as a final pipeline after a default pipeline, a pipeline run after
 a reroute processor, or pipelines in pipeline processors.
 
-`produced_in_bytes`::
+`produced_as_first_pipeline_in_bytes`::
```

Comment:
This name is worse than `ingested_as_first_pipeline_in_bytes` since the stat is counted at the end of the sequence of pipelines, which were initiated by this pipeline.

Reply:
I don't have any good suggestions for a name here :( the only thing I could think of is `data_generated_as_first_pipeline_in_bytes` or `data_produced_as_first_pipeline_in_bytes` which is still kind of "meh"

Reply:
Yeah ... man naming is hard. How about `data_produced_downstream_of_this_pipeline_which_was_first_of_sequence_in_bytes`  :grin: 

I guess I'll just go with `produced_as_first_pipeline_in_bytes`

Reply:
Yeah the goal here isn't so much to find a perfectly descriptive name as it is to find a name sufficiently complex that folks feel compelled to look up the exact meaning in the docs :)